### PR TITLE
Added duration flag for update period

### PR DESCRIPTION
**Pull request by oxisto**
I've named the flag "updateperiod", but i'm willing to change it to something more suitable.

### CODE REVIEW

File: cmd/sbserver/main.go [hunk 1]
```diff
@@ -220,9 +220,10 @@ const (
 )
 
 var (
-	apiKeyFlag   = flag.String("apikey", "", "specify your Safe Browsing API key")
-	srvAddrFlag  = flag.String("srvaddr", "localhost:8080", "TCP network address the HTTP server should use")
-	databaseFlag = flag.String("db", "", "path to the Safe Browsing database.")
+	apiKeyFlag       = flag.String("apikey", "", "specify your Safe Browsing API key")
+	srvAddrFlag      = flag.String("srvaddr", "localhost:8080", "TCP network address the HTTP server should use")
+	databaseFlag     = flag.String("db", "", "path to the Safe Browsing database.")
+	updatePeriodFlag = flag.Duration("updateperiod", safebrowsing.DefaultUpdatePeriod, "the update period for the Safe Browsing database in minutes.")
```
Comment (dsnet) [inline cmd/sbserver/main.go]: Remove " in minutes". The flag.Duration requires a time unit to be passed, so the following are all valid: "10s", "20m", "1h"

**Comment by shahbag** [comment]
The API sets a minimum update duration. [https://developers.google.com/safe-browsing/v4/request-frequency]. 30 minutes is a good default. I do not think we should add a flag to change this.

**Comment by dsnet** [comment]
Perhaps the logic should actually respect the `minimumWaitDuration` field in the response and update with `max(minimumWaitDuration, updatePeriod)`?

The downside to this is that the `updatePeriod` parameter now has caveats attached to its meaning and usage.

**Comment by alexwoz** [comment]
Agreed with @shahbag. Clients should always abide by the update interval returned by the server.

**Pull request closed without merging**